packages/hooks/src/useSendOTPVerificationCode.ts

```diff
@@ -36,22 +41,23 @@ const useSendOTPVerificationCode = () => {
 
     const formatEmailOtpError = (error: TSocketError<'phone_number_challenge'>['error']) => {
         switch (error.code) {
-            case 'ExpiredCode':
-                setPhoneOtpErrorMessage(localize('Code expired.'));
+            case 'EmailCodeExpired':
+                setPhoneOtpErrorMessage(localize('Code expired. Get a new code.'));
                 break;
             case 'InvalidToken':
-                setPhoneOtpErrorMessage(localize('Invalid code. Press the link below to get a new code.'));
+                setPhoneOtpErrorMessage(localize('Invalid code. Try again or get a new code.'));
                 break;
             case 'NoAttemptsLeft':
-                setPhoneOtpErrorMessage(localize('Invalid code. OTP limit reached.'));
+                setIsForcedToExitPnv(true);
+                setShowCoolDownPeriodModal(true);
```

Comment:
Clarification: What does setIsForcedToExitPnv do?

Reply:
So when the cool down period modal appears for user, user will click on `OK` button, we are trying to redirect to personal details, by having this function, we will not show `cancel-verification-modal` when redirection is happening

Reply:
got it . Thanks @suisin-deriv